src/legacy/ui/public/agg_types/buckets/terms.js

```diff
@@ -136,27 +136,6 @@ export const termsBucketAgg = new BucketAggType({
         orderAgg.id = termsAgg.id + '-orderAgg';
         return orderAgg;
       },
-      controller: function ($scope) {
-        $scope.$watch('responseValueAggs', updateOrderAgg);
-        $scope.$watch('agg.params.orderBy', updateOrderAgg);
-
-        function updateOrderAgg() {
```

Comment:
This logic was moved to `order_agg.tsx` control
https://github.com/elastic/kibana/pull/39502/files?file-filters%5B%5D=.tsx#diff-5f835f4980f47f394ea31d3b2bc86ec9R44